### DISCUSSION TITLE
Load Daily  giving wrong value

### DISCUSSION THREAD

**Discussion by kentishman2** [nathanmarlor/foxess_em#337] [Q&A]
How is Load Daily calculated? 
Its an AC1 5kV inverter with 17 panel on 3 sides with Enphase inverters
I have had to use a template to get the House energy consumption measured value.  Its a CT clamp around the tail going into the consumer unit.
![Screenshot_2023-09-28_20-22-24](https://github.com/nathanmarlor/foxess_em/assets/108495188/70257bbb-fb79-4f7a-9191-75c99ab50c76)

**Comment by nathanmarlor** [comment]
It pulls the last 48h of load power data and sums. Can you include a screenshot of the last 48h of the entity you used for load power when setting up EM please?

**Comment by kentishman2** [comment]
Thanks for the reply  looking at the one day and 2 day graphs I can see the problem 
When I first set up the template I had erroneous data see big spike on lower graph.
I think I need to wait one more day 
![image](https://github.com/nathanmarlor/foxess_em/assets/108495188/abc2db1d-a9e4-425b-b0d0-bb39526564a2)


![image](https://github.com/nathanmarlor/foxess_em/assets/108495188/19ba36c2-a9b9-40a9-8cfe-675b85dfa556)

**Reply by nathanmarlor** [reply]
Yep that'll do it, once you're 48h past that erroneous value the numbers should start to make more sense - let me know if not!
